### DISCUSSION TITLE
Getting 404 file not found when trying to publish

### DISCUSSION THREAD

**Discussion by mariajoafana** [CollectionBuilder/collectionbuilder.github.io#79] [General]
Hi! I'm not sure what I'm doing wrong but I'm getting this when I try to publish my collection. It says: 

404 File not found
The site configured at this address does not contain the requested file.
If this is your site, make sure that the filename case matches the URL.
For root URLs (like http://example.com/) you must provide an index.html file.

https://mariajoafana.github.io/inventar_colombia/
https://github.com/mariajoafana/inventar_colombia

Thanks!

**Comment by evanwill** [comment]
@mariajoafana your project is getting a error during Jekyll build, so it is not successfully deploying to github pages--which is why nothing is showing up! 

There three issues in your project with files that aren't formatted quite right which are each causing Jekyll to fail--they will all need to be fixed to get the site building:

First, the file ["metadata-template.csv" in your project](https://github.com/mariajoafana/inventar_colombia/blob/main/_data/metadata-template.csv) has the quoting messed on on line 13--because this results in an invalid CSV, it causes ruby to throw an error as it is processing the "_data" folder (even though you aren't using the file!). To fix, you can delete the file, *or* delete line 13, *or* fix the quoting on line 13. The whole file looks like some characters got corrupted, so you might just want to delete it.

Second, your ["_config.yml" ](https://github.com/mariajoafana/inventar_colombia/blob/main/_config.yml) isn't formatted quite correctly. To fix it: 
- on line 17 there is a space in front of `source-code` that needs to be deleted. 
- on line 28 the value of `description` starts with a quote `"` but doesn't have one at the end, so you need to add `"` at the end.
- on line 37 the value of your `metadata` doesn't match the filename of your actual csv in the "_data" folder. It currently says `inventarcolombia-metadata`, but should be switched to `metadata-inventarcolombia` to match your csv filename.

Third, you were customizing your ["browse-js.html" include](https://github.com/mariajoafana/inventar_colombia/blob/main/_includes/js/browse-js.html), in [commit 11130256](https://github.com/mariajoafana/inventar_colombia/commit/1113025620d59e5323607539fe85a24c42b98680). This commit removed the Liquid `{% endif %}`, which leaves the Liquid if expression unclosed--this also causes a build error. You should undo this change--after line 2 in the file, click enter, and re-add the line `{% assign icons = cb_icons.icons %}{%- endif -%}`.

Please let me know if that helps--if it is unclear, I can make a Pull Request fixing these issues for you--just let me know!
Good luck!

**Reply by mariajoafana** [reply]
Thank so much! I'll fix all of it and let you know
